app/src/main/java/com/example/cm/data/repositories/MeetupRepository.java

```diff
@@ -74,17 +75,39 @@ private void listenToMeetupListChanges() {
         return meetupMLD;
     }
 
+    public MutableLiveData<List<String>> getAllUsersForMeetup(String meetupId) {
+        meetupCollection.document(meetupId).addSnapshotListener(executorService, (value, error) -> {
+            if (error != null) {
+                return;
+            }
+            if (value != null && value.exists()) {
+                List<String> users = new ArrayList<>();
+                if (value.get("invitedFriends") != null) {
+                    users.addAll((List<String>) value.get("invitedFriends"));
```

Comment:
Extract these values "inivtedFriends", "confiremdFriends" and "declinedFriends" to Constants since they are used in multiple places